recirq/seniority_zero/circuits_expt/gates_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2023 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validate that the gates are correct for this experiment
+
+we will use the Givens rotation gate from Hartree-Fock experiment
+as a base
+"""
+import cirq
+import numpy as np
+
+from recirq.seniority_zero.circuits_expt.gates import (
+    cnot_from_cz,
+    cnot_from_sqrt_swap,
+    gsgate_from_cz,
+    gsgate_from_sqrt_swap,
+)
+
+
+def givens_gate(a, b, theta) -> cirq.Circuit:
+    """Implements the givens rotation with sqrt(iswap).
```

Comment:
Do we need a blanline here? I guess test functions aren't rendered in any docs.

Reply:
Easy enough to add :)